fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/join/ReorderJoinRule.java

```diff
@@ -93,7 +94,6 @@ void enumerate(JoinOrder reorderAlgorithm, OptimizerContext context, OptExpressi
             } else {
                 outputColumns.union(oldRoot.getProjection().getOutputColumns());
                 projectMap.putAll(oldRoot.getProjection().getColumnRefMap());
-                commonProjectMap.putAll(oldRoot.getProjection().getCommonSubOperatorMap());
```

Comment:
Why remove commonProjectMap?

Reply:
CommonProjectMap is unused in join reorder phase